db/migrate/20200902213827_add_chain_index_to_patients_three.rb

```diff
@@ -0,0 +1,5 @@
+class AddChainIndexToPatientsThree < ActiveRecord::Migration[6.0]
+  def change
+    add_index :patients, [:jurisdiction_id, :isolation, :purged, :assigned_user], name: 'patients_index_chain_three_1'
```

Comment:
What are your thoughts on an additional index chain like this one that includes `monitoring`? It is checked in all but the query for the "all", "transferred in" and "transferred out" tabs it looks like, unless I missed one. I understand there are memory implication (as well as update time implications) when adding indexes to tables, but given how frequently that is checked in addition to all of the above, it could be worth it.

Reply:
Hi @kylietmo , I was going based off of the slow queries I was seeing in new relic. I know the status scopes include the `monitoring` field, but it seems like those slow queries don't include it. Would you be able to find an example of one of the slow queries that contain the `monitoring` field. If we are to add it to the chain index, we need to know the correct order of fields

Reply:
Mmm, gotcha. I wasn't basing that off what new Relic showed exactly, but New relic has been saying that really the entire query executed in the Patients#show action is the main pain point so I was looking at how that query was built up before execution which often included a check for monitoring. Let's go with what you have for now and if we need to, we can go update in the future.